threem.py

```diff
@@ -131,13 +131,16 @@ def signature(self, method, path):
         signature = base64.b64encode(digest)
         return signature, now
 
-    def request(self, path, body=None, method="GET", identifier=None,
-                max_age=None):
+    def full_url(self, path):
```

Comment:
This is unrelated--it's needed by the circulation branch I'm working on to get 3M using this system.